src/TSMapEditor/UI/Windows/ScriptsWindow.cs

```diff
@@ -551,7 +652,12 @@ private void EditScript(Script script)
                 });
             }
 
+            ddScriptColor.SelectedIndex = ddScriptColor.Items.FindIndex(item => item.Text == editedScript.EditorColor);
+            if (ddScriptColor.SelectedIndex == -1)
```

Comment:
Just a personal preference, to be cleaner with the style of the rest of the codebase.

```suggestion
            if (ddScriptColor.SelectedIndex < 0)
```